### ISSUE TITLE
allGreenhouseJobPost only includes one post of each job

### ISSUE THREAD

**Issue by nonphoto** [diegolamanno/gatsby-source-greenhouse#12]
I'm trying to query all job posts across multiple boards. Specifically I have one job that has multiple posts with the same name, but `allGreenhouseJobPost` only includes one post of each job.

For example here's my query:
```{
  allGreenhouseJobPost {
    edges {
      node {
        job_id
        external
        title
      }
    }
  }
}
```

This returns only one job, but it should return two jobs with the same `job_id`: one internal and one external.

**Comment by joe-alfaro-anchorage** [comment]
PR #13 is my fix for this. You can use my fork and run locally if it doesn't get merged, we are currently using it on our site and it is working fine.

**Comment by diegolamanno** [comment]
Thanks @joe-alfaro-anchorage PR #13 has been merge and released with `v3.0.0`. Thanks
